template/package.json

```diff
@@ -30,9 +30,6 @@
     "react-test-renderer": "18.0.0",
     "typescript": "^4.4.4"
   },
-  "resolutions": {
-    "@types/react": "^18"
```

Comment:
You should probably keep this in considering that `@types/*` dependencies of `@types/*` always use `*` instead of pinning to SemVer MAJOR. At least until https://github.com/microsoft/DefinitelyTyped-tools/issues/433 is resolved.

For example, if we publish `@types/react@19.0.0` then any new install of this package.json (or rather installs without lockfile) will pull in `@types/react@19.0.0` instead of `^18`.

Reply:
@eps1lon isn't this PR resolving this issue? https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61081/files

Reply:
It opens up another though once `@types/react@19.0.0` is released:


```json
"dependencies": {
    "@types/react": "*"
},
```

-- https://unpkg.com/@types/react-native@0.69.3/package.json

Reply:
I don't think React 19 is coming any time soon ;)

Reply:
We are going to merge and update the types if needed after.

Reply:
I will remember this thread for the typescript compile error I get in 2023 :laughing: :hourglass_flowing_sand: